JeffShepherd/BaldEagles
# Feature/vertical character movement

## What does this PR do?

- Add canvas element and variable representing its context
- Add image to 
- Add function to render new game frame every 80ms
- Add variables representing gravity and apply gravity at an increasing rate
- Add listeners for spacebar depression and mouse click and decrease fall rate when activated

## How should it be tested in the UI?

- Run 'npm start' and ensure the canvas appears at approximately 500x400px with a light background and a small eagle
- The eagle should start falling immediately at a moderate pace
- The eagle should rise slightly after a click or spacebar depression, but not for anything else
- The eagle should start falling again after a click or spacebar depression
- Out of scope: eagle disappearing off screen as there is not a boundary set yet
